lib/SPIRV/SPIRVReader.cpp

```diff
@@ -2909,14 +2909,19 @@ Function *SPIRVToLLVM::transFunction(SPIRVFunction *BF) {
     mapValue(BA, &(*I));
     setName(&(*I), BA);
     BA->foreachAttr([&](SPIRVFuncParamAttrKind Kind) {
-      if (Kind == FunctionParameterAttributeNoWrite)
-        return;
       Attribute::AttrKind LLVMKind = SPIRSPIRVFuncParamAttrMap::rmap(Kind);
```

Comment:
Out of curiosity: I notice that FunctionParameterAttributeNoReadWrite is not listed in SPIRSPIRVFuncParamAttrMap. What happens here if the NoReadWrite function parameter attribute is seen?

I don't think NoReadWrite <-> readnone should be dealt with as part of this PR (assuming the mapping would be valid), but I would like to know.

Reply:
I think nothing will happen. Also this situation will happen if we try to translate a SPIR-V module created not by the translator, since it's not capable to generate `FunctionParameterAttributeNoReadWrite` yet.